docs/interactions/Application_Commands.md

```diff
@@ -280,11 +280,14 @@ Returned when fetching the permissions for a command in a guild.
 
```

Comment:
In the line above, could we reflect this change?

Maybe change to "Returned when fetching the permissions for a command or application in a guild"